monitoring/monitorlib/scd.py

```diff
@@ -125,3 +126,65 @@ def valid(self) -> bool:
   @property
   def version(self) -> Optional[int]:
     return self.get('version', None)
+
+
+################################################################################
+#################### Start of ASTM-standard definitions    #####################
+#################### interfaces/astm-utm/Protocol/utm.yaml #####################
+################################################################################
+
+class LatLngPoint(ImplicitDict):
+    '''A class to hold information about a location as Latitude / Longitude pair '''
+    lat: float
+    lng: float
+
+class Radius(ImplicitDict):
+    ''' A class to hold the radius of a circle for the outline_circle object '''
+    value: float
+    units: str
+
+class Polygon(ImplicitDict):
+    ''' A class to hold the polygon object, used in the outline_polygon of the Volume3D object '''
+    vertices: List[LatLngPoint] # A minimum of three LatLngPoints are required
+
+class Circle(ImplicitDict):
+    ''' A class the details of a circle object used in the outline_circle object '''
+    center: LatLngPoint 
+    radius: Radius
+
+class Altitude(ImplicitDict):
+    ''' A class to hold altitude information '''
+    value:float
+    reference:str
+    units: str 
+
+class Time(ImplicitDict):
+    ''' A class to hold Time details '''
+    value: StringBasedDateTime 
+    format:str 
```

Comment:
This is one place where `Literal['RFC3339']` may be appropriate (same with other constants like `Altitude.reference`), but the current type is not wrong.

Reply:
OK thanks, I have now updated both the formats.